lib/tasks/publishing_api.rake

```diff
@@ -241,9 +241,14 @@ namespace :publishing_api do
   end
 
   desc "Republish all HTML attachments to the Publishing API"
-  task republish_html_attachments: :environment do
-    HtmlAttachment.where(attachable_type: "Edition", attachable_id: Edition.publicly_visible.select("id")).each do |a|
-      Whitehall::PublishingApi.bulk_republish_async(a)
+  task :republish_html_attachments, [:state] => :environment do |_, args|
+    args.with_defaults(state: "withdrawn")
+    publications = Publications.where(state: args[:state])
```

Comment:
It looks like this lacks the `publicly_visible` part of the previous iteration. Does it need this? I'm not sure if this restriction is needed prior to going to republish worker